README.md

```diff
@@ -128,3 +129,16 @@ If you use [saleor-dashboard](https://github.com/saleor/saleor-dashboard) and yo
 Commands `build` and `dev` would generate schema and typed functions using Saleor's GraphQL endpoint. Commit the `generated` folder to your repo as they are necessary for queries and keeping track of the schema changes.
 
 [Learn more](https://www.graphql-code-generator.com/) about GraphQL code generation.
+
+### Storing registration data - APL
+
+During registration process Saleor API pass the auth token to the app. With this token App can query Saleor API with privileged access (depending on requested permissions during the installation).
+To store this data, app-template use a different [APL interfaces](https://github.com/saleor/saleor-app-sdk/blob/main/docs/apl.md).
+
+The choice of the APL is done using `APL` environment variable. If value is not set, FileAPL is used. Available choices:
+
+- `file`: no additional setup is required. Good choice for local development. Can't be used for multi tenant-apps or be deployed (not intended for production)
+- `upstash`: use [Upstash](https://upstash.com/) Redis as storage method. Free account required. Can be used for development and production and supports multi-tenancy
+- `vercel`: used by deployments from the Marketplace. It's single-tenant only and only supported by Vercel deployments done with Saleor CLI
```

Comment:
Maybe you can add example?
```
// .env

APL=vercel
SALEOR_REGISTER_APP_URL=...
SALEOR_DEPLOYMENT_TOKEN=...
```

It will highlight that each APL requires some extra envs

Reply:
Not sure if should duplicate docs in two separate places - this readme and APL. IMO link to SDK docs is ok since it's the most up-to-date APL documentation.